server/util/fetch.js

```diff
@@ -1,12 +1,10 @@
 const minipassFetch = require('make-fetch-happen');
-const https = require('https');
```

Comment:
Can the `https` npm package be removed? I don't see it being used directly elsewhere.

Reply:
Yes it can, and should! I'll add a commit to remove it.